app/design/frontend/Magento/luma/Magento_Theme/web/css/source/_module.less

```diff
@@ -363,6 +363,21 @@
 //  _____________________________________________
 
 .media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__m) {
+    html,
+    body {
+        height: 100%; // Stretch screen area for sticky footer
```

Comment:
Use `height: 100%` is a bad practice because content can be more than 100% and in this case, body block will not adaptive to content, it will be always 100% independent to content, that's is wrong.

Reply:
@VladimirZaets  
I have checked with content and without content page I don't see any issue .

Can you see attached image.
If still you have any query then you attach issue screen shot. I will try to solve
![with-content](https://user-images.githubusercontent.com/15214167/40549214-7d1332a8-6054-11e8-8428-87cb28dccede.png)
![without-content png](https://user-images.githubusercontent.com/15214167/40549217-7d5b84ae-6054-11e8-85bb-65b4d24e2e13.png)

Reply:
But in Safari and other browsers and cases. 'min-height: 100%` should be used.

Reply:
> If i will use 'min-height' then I can see some responsive issue.

Please elaborate what issue you see.

Reply:
After add 'min-height:100% you can see issue in screenshot  ' footer is not touch in bottom.